frontend/public/vendor.scss

```diff
@@ -40,7 +40,6 @@
 @import '~patternfly/dist/sass/patternfly/popovers';
 @import '~patternfly/dist/sass/patternfly/forms';
 @import "~xterm/dist/addons/fullscreen/fullscreen";
-@import "~patternfly-react/dist/sass/field-level-help";
```

Comment:
attn:  @mareklibra, @vojtechszocs This will impact kubevirt and you'll need to add this to https://github.com/openshift/console/blob/master/frontend/packages/kubevirt-plugin/src/style.scss

Reply:
Nevermind.  I see that kubevirt is pulling in *all* of patternfly-react SASS.  :/